CHANGELOG.md

```diff
@@ -0,0 +1,77 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
+### Added
+
+-
+
+### Fixed
+
+-
+
+## [2.1.0] - 2021-03-25
+
+### Changed
+
+- init.sh should be used
```

Comment:
We are not ready for an auto-import from commit messages. For now, we need manually aggregate changes into the understandable list of changes (1, 2, 3, ...)